src/bin/rad-ci.rs

```diff
@@ -0,0 +1,161 @@
+use std::convert::TryInto;
+use std::io::{Read, Write};
+use std::net::TcpStream;
+
+use rad_ci::{Options, HELP};
+use rad_terminal::components as term;
+
+use anyhow::Context;
+use ssh2::Session;
+
+fn main() {
+    rad_terminal::args::run_command::<Options, _>(HELP, "CI", run);
+}
+
+fn run(options: Options) -> anyhow::Result<()> {
```

Comment:
This should be moved to the `lib.rs`, along with any dependencies

Reply:
https://github.com/radicle-dev/radicle-client-tools/pull/35/files#diff-2f5300f7dedf306f4b73a94a6c5c3674f3ece69ed4bda39a3b27b8448149d978R119